### PR TITLE
Including 10 new regression tests to test suite.

### PR DESCRIPTION
**Pull request by tskille**
10 new regression test are included in the test suite. All of these are based on model2 in opm-test. Model2 was merged into opm-test with [PR#77](https://github.com/OPM/opm-tests/pull/77)

### CODE REVIEW

**Comment by akva2** [comment]
missing updates to tests/update_reference_data.sh

**Comment by tskille** [comment]
All data files related to model2 has been renamed => PR #135 in opm-tests. This should be merged before merging this. 

All tests related to model2 has been run locally towards. script update_reference_data.sh has not been tested. Is there a way that I can test this myself locally? If so, please let me know.

File: tests/update_reference_data.sh [hunk 1]
```diff
@@ -52,22 +52,50 @@ tests[spe1_metric_vfp1]="flow vfpprod_spe1 SPE1CASE1_METRIC_VFP1 spe1_metric_vfp
...
 for test_name in ${!tests[*]}
 do
+  
   binary=`echo ${tests[$test_name]} | awk -F ' ' '{print $1}'`
   dirname=`echo ${tests[$test_name]} | awk -F ' ' '{print $2}'`
   casename=`echo ${tests[$test_name]} | awk -F ' ' '{print $3}'`
   tname=`echo ${tests[$test_name]} | awk -F ' ' '{print $4}'`
+
+
+  echo $test_name
```
Comment (akva2) [inline tests/update_reference_data.sh]: hmm

File: tests/update_reference_data.sh [hunk 2]
```diff
@@ -52,22 +52,50 @@ tests[spe1_metric_vfp1]="flow vfpprod_spe1 SPE1CASE1_METRIC_VFP1 spe1_metric_vfp
...
+  echo $dirname
+  echo $casename
+  echo $tname
+
+
   test -z "$tname" && tname=$dirname
+  
+  echo $configuration
+   
+  exit 1
```
Comment (akva2) [inline tests/update_reference_data.sh]: hmm

File: tests/update_reference_data.sh [hunk 3]
```diff
@@ -52,22 +52,50 @@ tests[spe1_metric_vfp1]="flow vfpprod_spe1 SPE1CASE1_METRIC_VFP1 spe1_metric_vfp
...
+
   copyToReferenceDir \
       $configuration/build-opm-simulators/tests/results/$binary+$tname/ \
       $OPM_TESTS_ROOT/$dirname/opm-simulation-reference/$binary \
       $casename \
       EGRID INIT RFT SMSPEC UNRST UNSMRY
+  
+  
+  exit 1
```
Comment (akva2) [inline tests/update_reference_data.sh]: hmm

Comment (tskille) [reply tests/update_reference_data.sh]: Sorry about this mess. I was trying to understand how the script works and then I added some extra lines and tried to run this. Unfortunately I forgot to remote this.

**Comment by akva2** [comment]
jenkins build this update_data please

File: tests/update_reference_data.sh [hunk 4]
```diff
@@ -52,6 +52,16 @@ tests[spe1_metric_vfp1]="flow vfpprod_spe1 SPE1CASE1_METRIC_VFP1 spe1_metric_vfp
 tests[base_model_1]="flow model1 BASE_MODEL_1 base_model_1"
 tests[msw_model_1]="flow model1 MSW_MODEL_1 msw_model_1"
 tests[faults_model_1]="flow model1 FAULTS_MODEL_1 faults_model_1"
+tests[base_model2]="flow model2 0_BASE_MODEL2 base_model_model2"
```
Comment (akva2) [inline tests/update_reference_data.sh]: wrong testname. s/base_model_model2/base_model2/g

**Comment by akva2** [comment]
jenkins build this update_data please

File: tests/update_reference_data.sh (hunk 4 above)
Comment (tskille) [reply tests/update_reference_data.sh]: thanks, fixed

**Comment by akva2** [comment]
jenkins build this opm-tests=136 please

**Comment by akva2** [comment]
ready to be merged (preferably squashed) as far as i am concerned.

**Comment by akva2** [comment]
jenkins build this opm-tests=136 please

**Pull request merged**